.azure/pipelines/richnav.yml

```diff
@@ -6,58 +6,73 @@
 trigger:
   branches:
     include:
-    - blazor-wasm
     - main
     - release/*
-    - internal/release/*
+
+# Do not run this pipeline for PR validation.
+pr: none
```

Comment:
@jepetty @brianaeng is there any point in running this pipeline for PRs❔

Reply:
Benefit would be you could use github.dev to review PRs with Code Index enabled.

Since the repo is pretty hefty, let's start with just default branch, then can evaluate enabling PRs if more folks on the team use github.dev for PR review, or once we get native github.com integration for PRs

Reply:
Thanks @jepetty. I'll leave this as-is for now unless I hear the _only_ value developers perceive is for PR navigation. "As-is" means rich nav information is uploaded for 'main' and our servicing branches.